fb_apt/README.md

```diff
@@ -0,0 +1,92 @@
+fb_apt Cookbook
+====================
+The `fb_apt` cookbook installs and configures APT, the Debian package 
+management tool.
+
+Requirements
+------------
+
+Attributes
+----------
+* node['fb_apt']['config']
+* node['fb_apt']['keys']
+* node['fb_apt']['keyring']
+* node['fb_apt']['keyserver']
+* node['fb_apt']['mirror']
+* node['fb_apt']['preserve_sources_list_d']
+* node['fb_apt']['preferences']
+* node['fb_apt']['repos']
+* node['fb_apt']['update_delay']
+* node['fb_apt']['want_backports']
+* node['fb_apt']['want_non_free']
+* node['fb_apt']['want_source']
+
+Usage
+-----
+To install and configure APT include `fb_apt`, which will populate the 
+repository sources in `/etc/apt/sources.list` and update the package cache 
+every `node['fb_apt']['update_delay']` seconds (defaults to 86400) using the 
```

Comment:
see other comments. I think every run is fine. Also, if you do leave this feature the description is misleading. It's not every `update_delay` seconds, it's every run where the MD is older than `update_delay` seconds